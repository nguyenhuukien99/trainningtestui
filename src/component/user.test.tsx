import renderer from "react-test-renderer";
import User from "./user";
// Updated test case with a Link to a different address
it("renders correctly", () => {
  const tree = renderer
    .create(
      <User
        name="Nguyen Huu Kien"
        age={23}
        birthday="20/04/1999"
        src="https://znews-photo.zingcdn.me/w660/Uploaded/unvjuas/2021_01_12/106228561_288966105802103_4617392174817760819_n.jpg"
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
