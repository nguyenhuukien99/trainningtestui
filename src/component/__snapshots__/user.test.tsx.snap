// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-bcXHqe bBIowY"
>
  <div
    className="info"
  >
    <div
      className="text"
    >
      <div
        className="name"
      >
        Ho va ten:
        Nguyen Huu Kien
      </div>
      <div
        className="age"
      >
        Tuoi:
        23
      </div>
      <div
        className="birthday"
      >
        Ngay Sinh:
        20/04/1999
      </div>
    </div>
    <div
      className="image"
    >
      <img
        alt="nancy"
        src="https://znews-photo.zingcdn.me/w660/Uploaded/unvjuas/2021_01_12/106228561_288966105802103_4617392174817760819_n.jpg"
      />
    </div>
  </div>
</div>
`;
